.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
-max-complexity = 10
-max-line-length = 88
-extend-ignore = E203,C901,E501
+max-complexity = 101
```

Comment:
> NOTE: max-complexity should be 10, not 101! I have never seen a complexity level so high.

So now we need to refactor the code until this can be 10 again?

Reply:
A future work effort perhaps